packageManager.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
please separate this functionality into a packager and use this new package in the cordova-archive step